source/common/ssl/context_config_impl.h

```diff
@@ -12,16 +12,27 @@
 namespace Envoy {
 namespace Ssl {
 
+static const std::string INLINE_STRING = "<inline>";
```

Comment:
From an interface point of view, I'd rather have all these functions return empty string if they're inline, and then change that to this human-readable string right before it is presented to a user.  But I don't feel strongly about it.

Reply:
From an interface point of view, I think it's better to have this logic self-contained here, instead of intermixed with the code. I don't feel too strongly about it, but I'm going to leave it as-is.

Reply:
That's fine; if someone ever needs it different, it's easy enough to change then.